### PR TITLE
Added docs for CollectionProxy#take 

### PR DESCRIPTION
**Pull request by ronakjangir47**

### CODE REVIEW

File: activerecord/lib/active_record/associations/collection_proxy.rb [hunk 1]
```diff
@@ -227,6 +227,31 @@ def last(*args)
         @association.last(*args)
       end
 
+      # Gives a record (or N records if a parameter is supplied) from the collection
+      # Uses the same rules as <tt>ActiveRecord::Base.take</tt>.
```
Comment (eileencodes) [inline activerecord/lib/active_record/associations/collection_proxy.rb]: Thanks for the PR! :smile:

Is this supposed to be two sentences or one? If it's 2 can you add the period at the end of the first sentence? If it's 1 then lowercase uses and change it to using. Thanks!

Comment (ronakjangir47) [inline activerecord/lib/active_record/associations/collection_proxy.rb]: @eileencodes Sorry I missed it. Now updated

**Comment by eileencodes** [comment]
Thanks for the PR and updating it! :smile:

**Comment by ronakjangir47** [comment]
@eileencodes  Thanks for review

**Pull request merged**